TeaDiary/Views/Tea/Add.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@{
+    Layout = "/Views/Shared/_Layout.cshtml";
+}
+
+<body>
+<h2></h2>
```

Comment:
I think this page can use a title. Also, try using the same View for both Edit and Create.